src/applications/vaos/reducers/appointments.js

```diff
@@ -0,0 +1,33 @@
+import {
+  FETCH_APPOINTMENT_SUMMARY,
+  FETCH_APPOINTMENT_SUMMARY_SUCCEEDED,
+} from '../actions/appointments';
+
+const initialState = {
+  summary: {
+    loading: true,
+    data: null,
+  },
+};
+
+export default function appointmentsReducer(state = initialState, action) {
+  switch (action.type) {
+    case FETCH_APPOINTMENT_SUMMARY:
+      return {
+        ...state,
+        summary: {
```

Comment:
I think this will remove the `data` property, so it'll be undefined instead of null. But it looks like you're not using `data`, so we could remove it?

I don't think this would cause a problem, but I've seen some people advocate for not creating nested objects in your Redux state unless you really have to, since it can be confusing. I did that in my PR, but I don't have a strong opinion either way, as long as we're consistent.